src/tiledb/cloud/soma/ingest.py

```diff
@@ -136,15 +203,18 @@ def build_ingest_workflow_graph(
         )
 
         for entry_input_uri in vfs.ls(input_uri):
+            logging.info(f"ENUMERATOR ENTRY_INPUT_URI={entry_input_uri}")
             base = os.path.basename(entry_input_uri)
             base, _ = os.path.splitext(base)
 
             entry_output_uri = output_uri + "/" + base
             if not output_uri.endswith("/"):
                 entry_output_uri += "/"
             entry_output_uri += base
+            logging.info(f"ENUMERATOR ENTRY_OUTPUT_URI={entry_output_uri}")
 
             if pattern is not None and not re.match(pattern, entry_input_uri):
+                logging.info(f"ENUMERATOR SKIP NO MATCH ON <<{pattern}>>")
```

Comment:
`%r` in the log statement here means you'll get an escaped string

Reply:
Closing the loop on this @thetorpedodog :

```
>>> bar = "https://foo.com/bar?a=b&c=d"
>>> logging.info(f"foo {bar!r} baz")
INFO:root:foo 'https://foo.com/bar?a=b&c=d' baz
>>> logging.info("foo %r baz", bar)
INFO:root:foo 'https://foo.com/bar?a=b&c=d' baz
```

I am fine with this behavior.